Frontend/app/src/main/res/layout/activity_report.xml

```diff
@@ -5,6 +5,34 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:background="#4CAF50"
     android:orientation="vertical">
+    <RelativeLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content">
+
+        <ImageButton
+            android:id="@+id/deleteBtn"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="30dp"
+            android:layout_marginLeft="20dp"
+            android:layout_marginTop="30dp"
+            android:src="@android:drawable/ic_delete"
+            android:background="#00ff0000"
+            android:tint="@android:color/white"/>
+
+        <TextView
+            android:layout_toRightOf="@+id/deleteBtn"
```

Comment:
I think 'delete' means delete the report.
How about using 'close' to express to close report activity?